### ISSUE TITLE
Add README file

### ISSUE THREAD

**Issue by vbauer** [shelake-chetan/CustomToast#1]
It would be useful to add a minimal documentation in this repository

**Comment by shelake-chetan** [comment]
Yes sorry for that actually i am new on github and android arsenal. i have one question about android arsenal if i added Readme file at github repository it directly show on android arsenal or is it require to place manually?

**Comment by vbauer** [comment]
It will be updated automatically each few days
